<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog2</class>
 <widget class="QDialog" name="Dialog2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="group">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>329</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Hero</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Виробництво</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="plate_number">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>329</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="type_passage">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>440</y>
     <width>329</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="surname">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>329</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="uuid">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>329</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="sync">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>490</y>
     <width>329</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>True</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>False</string>
    </property>
   </item>
  </widget>
  <widget class="QDateTimeEdit" name="date">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>329</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="direction">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>329</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="name">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>329</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>111</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Surname</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>uuid</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>plate_number</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>group</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>date</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>direction</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>status</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>type_passage</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>sync</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="status">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>390</y>
     <width>329</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="send_data_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>560</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>ОК</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_window">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>560</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
